### PR TITLE
sys-cluster/{pacemaker-2.1.2,resource-agents-4.10.0}: fix command not found

### PR DESCRIPTION
**Pull request by scardracs**
- [x] Fix ocf_log: command not found
- [X] Bump resource-agents to 4.10.0

Closes: https://bugs.gentoo.org/720420
Closes: https://bugs.gentoo.org/830406
Closes: https://bugs.gentoo.org/830407

Package-Manager: Portage-3.0.30, Repoman-3.0.3
Signed-off-by: Marco Scardovi <marco@scardovi.com>

### CODE REVIEW

File: sys-cluster/resource-agents/resource-agents-4.10.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+RDEPEND="
+	sys-apps/iproute2
+	>=sys-cluster/cluster-glue-1.0.12-r1
+	libnet? ( net-libs/libnet:1.1 )
+"
+DEPEND="
+	${RDEPEND}
+	doc? (
+		dev-libs/libxslt
```
Comment (thesamesam) [inline sys-cluster/resource-agents/resource-agents-4.10.0.ebuild:26]: These are likely both BDEPEND (Libxslt provides binaries too).

**Review by thesamesam — APPROVED** [review]

File: sys-cluster/resource-agents/resource-agents-4.10.0.ebuild (hunk 1 above)
Comment (thesamesam) [reply sys-cluster/resource-agents/resource-agents-4.10.0.ebuild:26]: (Didn't mean to approve!)

Comment (scardracs) [reply sys-cluster/resource-agents/resource-agents-4.10.0.ebuild:26]: That PR is NOT ready yet, it's just to track my updates

File: sys-cluster/pacemaker/pacemaker-2.1.2.ebuild [hunk 2]
```diff
@@ -34,39 +34,40 @@ REQUIRED_USE="${PYTHON_REQUIRED_USE}"
...
-		--disable-fatal-warnings
-		--disable-static
-		--without-cs-quorum
-		--without-cman
-		--without-heartbeat
-		--with-corosync
-		--with-ais
-		$(use_with acl)
-		$(use_with smtp esmtp)
+	econf \
```
Comment (thesamesam) [inline sys-cluster/pacemaker/pacemaker-2.1.2.ebuild:47]: wait, really? Non-arrays are way more error prone (easier to miss a \) and it's generally considered cleaner to have an array :(

File: sys-cluster/pacemaker/pacemaker-2.1.2.ebuild [hunk 3]
```diff
@@ -34,39 +34,40 @@ REQUIRED_USE="${PYTHON_REQUIRED_USE}"
...
 }
 
 src_install() {
 	default
 	python_optimize
 
 	# remove provided initd file as we need support for OpenRC
-	rm -r "${ED}/etc/init.d" || die "Failed to remove old initd"
-	newinitd "${FILESDIR}/${PN}.initd" "${PN}"
+	# Without -f it fails to remove saying "File or directory not found"
```
Comment (thesamesam) [inline sys-cluster/pacemaker/pacemaker-2.1.2.ebuild:67]: It fails to remove _sometimes_, I think when systemd is installed, as it never installed the init.d in the first place.

If it always failed, we wouldn't bother having the line.

File: sys-cluster/pacemaker/files/pacemaker-2.1.2-command-not-found.patch [hunk 4]
```diff
@@ -0,0 +1,11 @@
+--- pacemaker-Pacemaker-2.1.2/configure.ac
```
Comment (thesamesam) [inline sys-cluster/pacemaker/files/pacemaker-2.1.2-command-not-found.patch:1]: change to a/, and b/, and link to the relevant bug(s) at the top of the patch

File: sys-cluster/resource-agents/resource-agents-4.0.1.ebuild [hunk 5]
```diff
@@ -1,59 +0,0 @@
-# Copyright 1999-2021 Gentoo Authors
```
Comment (thesamesam) [inline sys-cluster/resource-agents/resource-agents-4.0.1.ebuild:1]: Please don't rename the patch unless it's changing.

File: sys-cluster/pacemaker/pacemaker-2.1.2.ebuild [hunk 6]
```diff
@@ -54,19 +55,18 @@ src_configure() {
...
 }
 
 src_install() {
 	default
 	python_optimize
 
-	# remove provided initd file as we need support for OpenRC
-	rm -r "${ED}/etc/init.d" || die "Failed to remove old initd"
-	newinitd "${FILESDIR}/${PN}.initd" "${PN}"
+	# Remove provided initd file as we need support for OpenRC
```
Comment (thesamesam) [inline sys-cluster/pacemaker/pacemaker-2.1.2.ebuild:66]: You didn't address my comment yet btw

File: sys-cluster/pacemaker/pacemaker-2.1.2.ebuild (hunk 3 above)
Comment (scardracs) [reply sys-cluster/pacemaker/pacemaker-2.1.2.ebuild]: What should I have ti do?

File: sys-cluster/pacemaker/files/pacemaker-2.1.2-command-not-found.patch (hunk 4 above)
Comment (scardracs) [reply sys-cluster/pacemaker/files/pacemaker-2.1.2-command-not-found.patch]: Removed as it is not required

File: sys-cluster/pacemaker/pacemaker-2.1.2.ebuild (hunk 6 above)
Comment (scardracs) [reply sys-cluster/pacemaker/pacemaker-2.1.2.ebuild:66]: Which one?

File: sys-cluster/pacemaker/pacemaker-2.1.2.ebuild (hunk 3 above)
Comment (thesamesam) [reply sys-cluster/pacemaker/pacemaker-2.1.2.ebuild]: Just adjust the comment.

File: sys-cluster/pacemaker/pacemaker-2.1.2.ebuild (hunk 6 above)
Comment (thesamesam) [reply sys-cluster/pacemaker/pacemaker-2.1.2.ebuild:66]: https://github.com/gentoo/gentoo/pull/23636#discussion_r777686489

File: sys-cluster/resource-agents/resource-agents-4.0.1.ebuild (hunk 5 above)
Comment (scardracs) [reply sys-cluster/resource-agents/resource-agents-4.0.1.ebuild:1]: Oh, it was for convenience

**Pull request closed without merging**